app/jinja_filters.py

```diff
@@ -100,6 +105,20 @@ def format_conditional_date(date1=None, date2=None):
     return format_date(date)
 
 
+@blueprint.app_template_filter()
+def calculate_years_difference(date_from, date_to):
```

Comment:
if you make these parametrs `(from, to)`
then you don't need to suffiix `_converted` which looks messy